.github/workflows/gatk-tests.yml

```diff
@@ -0,0 +1,311 @@
+
+      - name: pull lfs files
+        run: git lfs pull
+
+      - name: run-docker-tests
+        id: docker-jacocco-tests
+        run: |
+          docker run \
+            -v $(pwd):/gatkCloneMountPoint:cached \
+            -v $(pwd)/testJars:/jars:cached \
+            --rm \
+            -e "scala.version=${SCALA_VERSION}" \
+            -e "TEST_VERBOSITY=minimal" \
+            -e "TEST_TYPE=${TEST_TYPE}" \
+            -t  ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ github.sha }} \
+            bash --init-file /gatk/gatkenv.rc /root/run_unit_tests.sh;
+          TEST_EXIT_VALUE=$?;
+          $( exit ${TEST_EXIT_VALUE} );
+          sudo chmod -R a+w build/reports/;
+          mkdir build/reports/tests/test \
+           && cp -rp build/reports/tests/testOnPackagedReleaseJar/* build/reports/tests/test \
+           && rm -r build/reports/tests/testOnPackagedReleaseJar;
+
+      - uses: /broadinstitute/gatk/.github/actions/upload-results@lb_github_actions
+        if: always()
+        with:
+          warnPR: ${{ github.event_name == 'pull_request' && steps.docker-jacocco-tests.outcome != 'success' }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          jdk-version: ${{ matrix.Java }}
+          test-type: docker ${{ matrix.testType }}
+          job-matrix-id: ${{ github.run_id }}.${{ strategy.job-index }}
+          repo-path: ${{ github.ref_name }}_${{ github.run_id }}.${{ strategy.job-index }}
+          bot-comment-key: ${{ secrets.GATK_BOT_COMMENT_KEY }}
+          identifier: Java ${{ matrix.Java }} test on docker ${{matrix.testType}}
+
+
+  wdlGen:
+    runs-on: ubuntu-latest
+    name: Test WDL Generation
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch: 0
+      - name: Set up java 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+          cache: gradle
+      - uses: /broadinstitute/gatk/.github/actions/install-cromwell@lb_github_actions
+        with:
+          CROMWELL_VERSION: ${{ env.CROMWELL_VERSION }}
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      - name: Run WDL Generation Test
+        run: ./gradlew gatkWDLGenValidation
+
+  wdlTests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        wdlTest: [ 'RUN_CNV_GERMLINE_COHORT_WDL', 'RUN_CNV_GERMLINE_CASE_WDL', 'RUN_CNV_SOMATIC_WDL', 'RUN_M2_WDL', 'RUN_CNN_WDL' ]
+    continue-on-error: true
+    name: WDL test ${{ matrix.wdlTest }} on cromwell
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: pull lfs files
+        run: git lfs pull
+      - name: Set up java 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+          cache: gradle
+      - uses: /broadinstitute/gatk/.github/actions/install-cromwell@lb_github_actions
```

Comment:
These branch identifiers are going to break.  Isn't there a way to refer to "this branch"?